Adds clarification about required endpoints for various situations. (#745)
commerce-docs/microsite-commerce-storefront: src/content/docs/setup/configuration/commerce-configuration.mdx

Reviewer: ```suggestion
<TabItem label="Adobe Commerce as a Cloud Service (ACCS)">
```

</TabItem>

<TabItem label="Adobe Commerce Optimizer (ACO)">

> This example shows the configuration for Adobe Commerce Optimizer (ACO), which uses different header naming conventions and includes Adobe Commerce Optimizer-specific settings. For ACO you must set **`commerce-endpoint`** (ACO GraphQL URL for catalog) at a minimum, and **`commerce-core-endpoint`** (your core endpoint—PaaS or your own Adobe Commerce hosted environment) if you want to use transactional or other core related functionality. You must also include `"adobe-commerce-optimizer": true`. This example is from https://main--boilerplate-aco--adobe-commerce.aem.live/ (no core endpoint).

Reviewer: ```suggestion
This example shows the configuration for Adobe Commerce Optimizer (ACO), which uses different header naming conventions and includes ACO-specific settings. For ACO, set commerce-endpoint (ACO GraphQL URL for catalog) at a minimum. Set commerce-core-endpoint (your core endpoint—PaaS or your own Adobe Commerce hosted environment) to enable transactional or other core-related functionality. Include "adobe-commerce-optimizer": true. This example is from https://main--boilerplate-aco--adobe-commerce.aem.live/ and does not include a core endpoint.
```

```json title="ACO config.json" showLineNumbers
{
  "public": {
    "default": {

>       "adobe-commerce-optimizer": true,

Author: This is a missed prop that is mandatory.
